### DISCUSSION TITLE
Problem accessing my local network

### DISCUSSION THREAD

**Discussion by richardolan** [pocketbase/pocketbase#717] [Q&A] [answered]
I am running pocketbase on Windows and it is impossible to access through my network, is something wrong?
 Example: http://192.168.1.42:8090/api/collections/demo/records

Error: This site cannot be reached192.168.2.155 connection refused.
Try:

Check the connection.
Check proxy and firewall.
ERR_CONNECTION_REFUSED

**Comment by ganigeorgiev** [comment, marked as answer]
Are you sure that your localhost is accessible from other devices in your network? I'm not familiar with windows, but on linux incoming connections are usually restricted by default.

Do you use your network IP when starting the PocketBase executable? If not try `./pocketbase serve --http="YOU_IP:8090"` (alternatively, you can also try binding to `0.0.0.0:8090` for all ipv4 interfaces).
